src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -320,11 +321,19 @@ private void MethodChecks(MethodDeclarationSyntax syntax, Binder withTypeParamsB
 
                     if ((object)overriddenMethod != null)
                     {
-                        CustomModifierUtils.CopyMethodCustomModifiers(overriddenMethod, this, out _lazyReturnType, 
-                                                                      out _lazyCustomModifiers, 
+                        CustomModifierUtils.CopyMethodCustomModifiers(overriddenMethod, this, out _lazyReturnType,
+                                                                      out _lazyCustomModifiers,
                                                                       out _lazyParameters, alsoCopyParamsModifier: true);
                     }
                 }
+                else if (_refKind == RefKind.RefReadOnly && (this.IsVirtual || this.IsAbstract))
+                {
+                    var isConst = withTypeParamsBinder.GetWellKnownType(WellKnownType.System_Runtime_CompilerServices_IsConst, diagnostics, syntax);
```

Comment:
Yes. I've a test that specifically looks for that. Check `ProperErrorsArePropagatedIfMscorlibIsConstIsNotAvailable` in `IsConstModifierTests.cs`.
The method `Binder.GetWellKnownType()` will report such errors to the passed diagnostics bag.

---
In reply to: [117828511](https://github.com/dotnet/roslyn/pull/19658#discussion_r117828511) [](ancestors = 117828511)